openshift/ocp_on_openstack.adoc

```diff
@@ -89,6 +96,176 @@ Push the `pass` changes to the git repository.
 git push
 ----
 
+== Post installation steps
+
+=== Install jump server
+
+The OCP cluster is created on a private network that allows communication 
```

Comment:
"The OCP cluster is created on a private network ..." --> "An OCP cluster, when created on RHOS", is deployed on a private OpenStack network ...

Reply:
changed